src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { stationsById } from "stations";
+import { JourneyParams } from "types";
+
+import styles from "./SuggestedJourneys.module.scss";
+
+interface journey {
+    // origin: string;
+    // destination: string;
+    // originStationId: JourneyParams["fromStationId"];
+    // destStationId: JourneyParams["toStationId"];
+    // day: JourneyParams["day"];
+    // time: JourneyParams["time"];
+    params: JourneyParams;
+    fromStationName: string;
+    toStationName: string;
+}
+
+type Props = { 
+    journeys: journey[];
+}
+
+export function getJourneyUrl(fromStationId: string, toStationId: string, day: string, time: number | undefined) {
```

Comment:
This function works well, but what if it took the `journey` (or `JourneyParams`) object as a single argument, since that has all of these properties on it? That would save you the trouble of referencing `journeys[i]` several times in the `for` loop below.

Minor note: there is no need to `export` this function since it doesn't get imported from other files.